seacrowd/sea_datasets/vimqa/vimqa.py

```diff
@@ -0,0 +1,187 @@
+import json
+import os
+from pathlib import Path
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """
+@inproceedings{le-etal-2022-vimqa,
+    title = "{VIMQA}: A {V}ietnamese Dataset for Advanced Reasoning and Explainable Multi-hop Question Answering",
+    author = "Le, Khang  and
+      Nguyen, Hien  and
+      Le Thanh, Tung  and
+      Nguyen, Minh",
+    editor = "Calzolari, Nicoletta  and
+      B{\'e}chet, Fr{\'e}d{\'e}ric  and
+      Blache, Philippe  and
+      Choukri, Khalid  and
+      Cieri, Christopher  and
+      Declerck, Thierry  and
+      Goggi, Sara  and
+      Isahara, Hitoshi  and
+      Maegaard, Bente  and
+      Mariani, Joseph  and
+      Mazo, H{\'e}l{\'e}ne  and
+      Odijk, Jan  and
+      Piperidis, Stelios",
+    booktitle = "Proceedings of the Thirteenth Language Resources and Evaluation Conference",
+    month = jun,
+    year = "2022",
+    address = "Marseille, France",
+    publisher = "European Language Resources Association",
+    url = "https://aclanthology.org/2022.lrec-1.700",
+    pages = "6521--6529",
+}
+"""
+
+_DATASETNAME = "vimqa"
+
+_DESCRIPTION = """
+VIMQA, a new Vietnamese dataset with over 10,000 Wikipedia-based multi-hop question-answer pairs. The dataset is human-generated and has four main features:
+The questions require advanced reasoning over multiple paragraphs.
+Sentence-level supporting facts are provided, enabling the QA model to reason and explain the answer.
+The dataset offers various types of reasoning to test the model's ability to reason and extract relevant proof.
+The dataset is in Vietnamese, a low-resource language
+"""
+
+_HOMEPAGE = "https://github.com/vimqa/vimqa"
+
+_LANGUAGES = ["vie"]
+
+_LICENSE = Licenses.UNKNOWN.value
```

Comment:
since they have EULA docs, prob the `LICENSE` value can refer to that doc
```suggestion
_LICENSE = f"""{Licenses.OTHERS.value} | \
    The licence terms for VimQA follows this EULA docs on their repo.
    Please refer to the following doc of EULA (to review the permissions and request for access)
    VIMQA EULA -- https://github.com/vimqa/vimqa/blob/main/VIMQA_EULA.pdf
"""
```